leaflet-ohm-timeslider.js

```diff
@@ -659,7 +659,7 @@ L.Control.OHMTimeSlider = L.Control.extend({
         //
         // warning: we are mutating someone else's map style in-place, and they may not be expecting that
         // if they go and apply their own filters later, it could get weird
-        const osmfilteringclause = [ 'any', ['>', '1', '0'] ];
+        const osmfilteringclause = ["==", "nodatefilter", ["literal", "nodatefilter"]];
```

Comment:
Is the idea that a style designer can opt a layer out of date filtering by wrapping its filter in this expression? If so, perhaps we should instead rely on some special property in the layer’s [`metadata`](https://maplibre.org/maplibre-style-spec/layers/#metadata) that we can document.

Reply:
The intent there is simply any expression that is always true. I was using `1 > 0` but `A == A` is just as good. My choice of the phrase "no date filter" is simply because no filter is in place yet as of line 662 - that comes later when the slider is manipulated.

Though I admit, the phrasing is apparently confusing. I can change it to `["==", 0, ["literal", 0]]` or `['>', '1', '0']` if you prefer.